src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -0,0 +1,46 @@
+package core.basesyntax;
+
+import core.basesyntax.animals.Cat;
+import core.basesyntax.animals.Dog;
+import java.time.LocalDate;
+
+public class HelloWorld {
+    private int date;
+    private int age;
+    private int size;
+    private String greetings;
+
+    public static void main(String[] args) {
+        Dog dog = new Dog();
+        Cat cat = new Cat();
+    }
+
+    private String getFromOneToTen() {
+        String result = "";
+        for (int i = 0; i < 10; i++) {
+            if (i % 2 == 0) {
+                result = result + i + " ";
+            }
+        }
+        return result;
+    }
+
+    private void sayHello(String hello) {
+        System.out.println("Hello, Mate! I am happy to see you! This class looks weird!"
+                + " Your task is to fix the checkstyle in this file."
+                + " Your goal is to run `mvn clean package` and receive"
+                + " `BUILD SUCCESS` message");
+        System.out.println(hello);
+    }
+
+    private void initializeVariables() {
+        greetings = "Hello mates!";
+        System.out.println(greetings);
+        date = LocalDate.now().getYear();
```

Comment:
I would say it would be more precise to give this variable `year` name, as it does not represent entire date, just year.